hy/core/shadow.hy

```diff
@@ -138,11 +138,17 @@
 (defn not-in [x y]
   (not-in x y))
 
+(defn get [thing key1 &rest keys]
+  (for* [k (+ (, key1) keys)]
+    (setv thing (get thing k)))
+  thing)
```

Comment:
Not the version I thought I approved.

Okay so why split off the first key? Is it for performance in the more common one-key case? In that case it should probably be
```Hy
(defn get [thing key1 &rest keys]
  (setv thing (get thing key1)
  (if keys
    (for* [k keys)]
      (setv thing (get thing k)))
    thing))
```
?
But complications due to performance optimizations like this need to be profiled to make sure they actually help.